Multisig/UI/WalletConnect/WalletConnectClientController.swift

```diff
@@ -13,6 +13,10 @@ import WalletConnectSwift
 class WalletConnectClientController {
     static let shared = WalletConnectClientController()
 
+    // We use specific key prefix to identify that this is our connection URL
+    // and not to connect to if it is in the Pasteboard on entering foreground
+    static let safeKeyPrefix = "gnosissafe".data(using: .utf8)!.toHexString()
```

Comment:
not sure about this one. Would cleaning up the pasteboard after first url detection solve the problem?

Reply:
The use case is:
1) user shows qr code to connect a 3rd party wallet. 
2) user presses "Copy to clipboard"
==> Safe to Wallet connection url in the pasteboard
3) user navigates to background
4) user navigates to foreground
==> Safe should not try to connect to itself

Reply:
ah ok